setup.py

```diff
@@ -10,16 +10,17 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6'],
     license='MIT',
-    author='zfrenchee',
-    author_email='alex@lenail.org',
+    author='zfrenchee, iamjli',
+    author_email='alex@lenail.org, iamjli@mit.edu',
```

Comment:
Please make sure setuptools can parse comma-delimited lists.

Reply:
[It can](https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py)